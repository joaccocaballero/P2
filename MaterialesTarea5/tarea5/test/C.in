insertarEnAbb (50,0)
insertarEnAbb (30,0)
insertarEnAbb (45,0)
insertarEnAbb (40,0)
insertarEnAbb (70,0)
insertarEnAbb (60,0)
insertarEnAbb (65,0)
rotar 30 LL
rotar 70 RR
imprimirAbb
rotar 70 LR
rotar 30 RL
imprimirAbb
rotar 40 RL
rotar 40 LR
imprimirAbb
rotar 50 LL
imprimirAbb
rotar 40 RR
imprimirAbb
Fin