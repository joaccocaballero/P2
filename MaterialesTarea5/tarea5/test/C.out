1>Se insertó (50,0.00) en el abb.
2>Se insertó (30,0.00) en el abb.
3>Se insertó (45,0.00) en el abb.
4>Se insertó (40,0.00) en el abb.
5>Se insertó (70,0.00) en el abb.
6>Se insertó (60,0.00) en el abb.
7>Se insertó (65,0.00) en el abb.
8>
9>
10>
-(70,0.00)
---(65,0.00)
--(60,0.00)
(50,0.00)
--(45,0.00)
---(40,0.00)
-(30,0.00)
11>
12>
13>
--(70,0.00)
-(65,0.00)
--(60,0.00)
(50,0.00)
--(45,0.00)
-(40,0.00)
--(30,0.00)
14>
15>
16>
--(70,0.00)
-(65,0.00)
--(60,0.00)
(50,0.00)
--(45,0.00)
-(40,0.00)
--(30,0.00)
17>
18>
---(70,0.00)
--(65,0.00)
---(60,0.00)
-(50,0.00)
--(45,0.00)
(40,0.00)
-(30,0.00)
19>
20>
--(70,0.00)
-(65,0.00)
--(60,0.00)
(50,0.00)
--(45,0.00)
-(40,0.00)
--(30,0.00)
21>Fin.
